cmd/minikube/cmd/cache.go

```diff
@@ -67,7 +67,7 @@ var deleteCacheCmd = &cobra.Command{
 }
 
 // LoadCachedImagesInConfigFile loads the images currently in the config file (minikube start)
-func LoadCachedImagesInConfigFile() error {
+func LoadCachedImagesInConfigFile(load bool) error {
```

Comment:
Having a verb in both the function name and the variable name is a sign that this should be refactored. Given the function name and docstring, I don't understand what this would actually do if load is false.

Would it help to split this into two functions? Perhaps just a "Images" function that returns a list of images, and the caller can decide if they should be loaded or cached?

Reply:
Two different functions now, similar to how othe things handle Cache and Load (here: Copy)